nomad/structs/structs_test.go

```diff
@@ -5188,22 +5188,26 @@ func TestPlan_AppendPreemptedAllocAppendsAllocWithUpdatedAttrs(t *testing.T) {
 	assert.Equal(t, expectedAlloc, appendedAlloc)
 }
 
-func TestAllocation_MsgPackTags(t *testing.T) {
+func TestMsgPackTags(t *testing.T) {
```

Comment:
Non-blocking: I realize you're just expanding a test we've already got, but I sort of wonder at the utility of this test. There's no logic under test, so isn't this basically just asserting that the struct is how we defined it?

Reply:
yeah I thought it was a little strange, but I assume it was added initially as a regression test, to avoid someone mistakenly deleting the field, or perhaps typoing the otherwise not-checked-by-anything magic string?

it also serves as a central inventory of structs with this tag (if the list is kept up to date), but a pretty simple grep can find that, too (which is how I found the previously untested ones).

so, meh, these almost-good reasons in aggregate were enough for me to keep the test around. :shrug:

Reply:
Works for me. It's a very cheap test anyways.